app.js

```diff
@@ -17,7 +17,6 @@ app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
```

Comment:
Just add a static middleware so the static files can be served...
You had mistakenly removed everything

```app.use(express.static(path.join(__dirname, "public")));```

Reply:
Done 👍